cmd/juno/dbcmd_test.go

```diff
@@ -0,0 +1,52 @@
+package main_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/NethermindEth/juno/blockchain"
+	"github.com/NethermindEth/juno/clients/feeder"
+	juno "github.com/NethermindEth/juno/cmd/juno"
+	"github.com/NethermindEth/juno/core"
+	"github.com/NethermindEth/juno/db/pebble"
+	adaptfeeder "github.com/NethermindEth/juno/starknetdata/feeder"
+	"github.com/NethermindEth/juno/utils"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+var emptyCommitments = core.BlockCommitments{}
+
+func TestDBCmd(t *testing.T) {
+	t.Run("retrieve info when db contains block0", func(t *testing.T) {
+		cmd := juno.DBInfoCmd()
+		executeCmdInDB(t, cmd)
+	})
+
+	t.Run("inspect db when db contains block0", func(t *testing.T) {
+		cmd := juno.DBSizeCmd()
+		executeCmdInDB(t, cmd)
+	})
+}
+
+func executeCmdInDB(t *testing.T, cmd *cobra.Command) {
+	cmd.Flags().String("db-path", "", "")
+
+	client := feeder.NewTestClient(t, &utils.Mainnet)
+	gw := adaptfeeder.New(client)
+	block0, err := gw.BlockByNumber(context.Background(), 0)
+	require.NoError(t, err)
+
+	stateUpdate0, err := gw.StateUpdate(context.Background(), 0)
+	require.NoError(t, err)
+
+	dbPath := t.TempDir()
+	testDB, _ := pebble.New(dbPath)
```

Comment:
do we need to check for error here?

Reply:
btw why can't we use `pebble.NewMemTest(...)` here ?

Reply:
since we are testing the CLI, we need an actual db path that can be resolved and opened.